pkgs/applications/audio/waon/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, fftw, gtk2, libao, libsamplerate
+, libsndfile, ncurses, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "waon";
+  version = "0.11";
+
+  src = fetchFromGitHub {
+    owner = "kichiki";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1xmq8d2rj58xbp4rnyav95y1vnz3r9s9db7xxfa2rd0ilq0ps4y7";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ fftw gtk2 libao libsamplerate libsndfile ncurses ];
+
+  installPhase = ''
+    install -Dt $out/bin waon pv gwaon
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A Wave-to-Notes transcriber";
+    homepage = https://kichiki.github.io/WaoN/;
```

Comment:
Please use quotas.

Reply:
We don't have a style guide requirement for this, and I'm yet to see anyone calling for deprecation of this syntax. At most, I'd drop the trailing slash for style reasons.

Reply:
Why? There are many other places that use non-quoted URLs, the style guide does not say it is disallowed, and it seems very weird to not use the feature if it exists.